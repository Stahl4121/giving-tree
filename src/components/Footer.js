import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import Container from '@material-ui/core/Container';
import Link from '@material-ui/core/Link';
import Grid from '@material-ui/core/Grid';
import Box from '@material-ui/core/Box';

function Copyright() {
  return (
   <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <Link color="inherit" href="https://material-ui.com/">
        Giving Tree
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  );
}

const useStyles = makeStyles(theme => ({
  footerRoot: {
    align: 'center',
    minHeight: '10vh',
  },
  footer: {
    padding: theme.spacing(3, 2),
    marginTop: 'auto',
    align: 'center',
    alignItems: 'center',
    backgroundColor: theme.palette.grey[300],
  },
}));

export default function Footer() {
  const classes = useStyles();
  return (
    <div className={classes.footerRoot}>
        <Container className={classes.footer}>
            <Copyright />
        </Container>
    </div>
  );
}