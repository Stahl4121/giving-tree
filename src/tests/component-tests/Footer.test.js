import React from "react";
import { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Footer from "../../components/Footer.js";

describe('Component: Footer', () => {
  var enzyme = require('enzyme');
  enzyme.configure({ adapter: new Adapter() });

  it('Renders correctly', () => {
    const wrap = enzyme.mount(<Footer/>)
    expect(wrap).toMatchSnapshot();
  });

})
