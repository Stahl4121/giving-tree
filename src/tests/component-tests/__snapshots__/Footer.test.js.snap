// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer Renders correctly 1`] = `ReactWrapper {}`;
